<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Login Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="1.207">
  <properties />
            <testcase name="VerifyThatUserShouldBuildYouOwnComputerAndAddThemToCartSuccessfully" time="0.000" classname="com.nopcommerce.demo.testsuite.ComputerTest">
                <failure type="org.testng.TestNGException"
                  message="
Method public void com.nopcommerce.demo.testsuite.ComputerTest.VerifyThatUserShouldBuildYouOwnComputerAndAddThemToCartSuccessfully(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : Build Your Own computer in class resources.testdata.TestData"
              >
<![CDATA[org.testng.TestNGException: 
Method public void com.nopcommerce.demo.testsuite.ComputerTest.VerifyThatUserShouldBuildYouOwnComputerAndAddThemToCartSuccessfully(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : Build Your Own computer in class resources.testdata.TestData
  org.testng.internal.Parameters.findDataProvider(Parameters.java:516)
  org.testng.internal.Parameters.handleParameters(Parameters.java:739)
  org.testng.internal.Parameters.handleParameters(Parameters.java:704)
  org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:70)
  org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:49)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:904)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.util.ArrayList.forEach(ArrayList.java:1249)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="verifyUserShouldNavigateToComputerPageSuccessfully" time="0.418" classname="com.nopcommerce.demo.testsuite.ComputerTest" />
                <testcase name="verifyUserShouldNavigateToDesktopsPageSuccessfully" time="0.789" classname="com.nopcommerce.demo.testsuite.ComputerTest" />
      
</testsuite>
